src/xdocs/config_coding.xml

```diff
@@ -4028,15 +4028,46 @@ for (String line: lines) {
         <source>
 &lt;module name=&quot;NestedForDepth&quot;/&gt;
         </source>
+        <p>Example: </p>
+        <source>
+for(int i=0; i&lt;10; i++) {
+    for(int j=0; j&lt;i; j++) {
+        for(int k=0; k&lt;j; k++) { // violation
```

Comment:
`// violation` -> `// violation, max allowed nested loop number is 1`

Reply:
fixed